### DISCUSSION TITLE
Problems with prettier and eslint

### DISCUSSION THREAD

**Discussion by matheus1lva** [LazyVim/LazyVim#1876] [Q&A]
Currently i have:
![CleanShot 2023-10-25 at 18 42 29](https://github.com/LazyVim/LazyVim/assets/831308/a258be74-930b-4789-806e-1986dea7e598)

And anytime i press save, it kind of doesnt format and eslint still complains and sometimes prettier and eslint goes into a battle and the files dont get formatted. How can i fix this?

**Comment by LuisFilipePedroso** [comment]
Have you added the eslint config into your LSP config?

**Comment by radvil** [comment]
remove all ur none-ls config, if u already have conform.nvim and nvim-lint setup, and dont forget to try reinstall ur prettier and eslint server via mason, so u hv a clean setup to reproduce
